### ISSUE TITLE
Suggestion : Graphical display of transition effect, calculate and display feed forward amount

### ISSUE THREAD

**Issue by ctzsnooze** [betaflight/betaflight-configurator#1109]
Few people understand the way in which the D weight (stick feed forward) factor is attenuated in relation to stick angle.

The absolute amount of the feed forward effect (PID 'boost' derived from rate of change of sticks) depends on the amount of D itself, as well as the D weight.  The absolute amount of that boost could be represented as D * weight (where the weight of 1 equals 100 CLI units).  For example, if someone had D at 35 and weight of 1, that could be considered to be 35 feed forward units.  And if someone had D at 20 and weight of 0.5, that could be considered as 10 feed forward units.  

A simple graphic could be developed, like the 'rates' graphic, to visualise the absolute amount of feed forward effect and the influence of stick position on that feed forward.

I think a graphic might much better explain what is happening to the feed forward effect when people change D, D weight, and transition, rather than just the numbers.

Overall colour and concept would be as per the current rates graphic.

Horizontal axis would be stick angle, and vertical axis would be feed forward units, defaulting to at 0-100.

Examples:

D of 25, weight 1.0, transition 0.25:
![d25_weight100_transition25](https://user-images.githubusercontent.com/11737748/42606754-9ea5d2ac-85c2-11e8-9794-a8524114ceef.jpg)

D of 34, weight 1.0, transition 0.5:
![d34_weight100_transition50](https://user-images.githubusercontent.com/11737748/42606770-af09d9ae-85c2-11e8-9b6a-eab233b4dedc.jpg)

D of 25, weight 1.0, transition 0 (or D of 50, weight .5, transition 0):
![d25_weight100_transition10](https://user-images.githubusercontent.com/11737748/42606799-d18052c4-85c2-11e8-984a-33d2e2958bb4.jpg)

D of 34, weight 1.0, transition 0:
![d34_weight100_transition0](https://user-images.githubusercontent.com/11737748/42606779-bdccb59c-85c2-11e8-8ac8-451918effb06.jpg)

D of 34, weight 1.0, transition 0.1
![d34_weight100_transition10](https://user-images.githubusercontent.com/11737748/42606831-f5aeac68-85c2-11e8-8e11-3eb0707907f1.jpg)
:

**Comment by McGiverGim** [comment]
Good idea. Maybe we need first to change a little the PID page to make space for this.
